playbooks/multi_node_aio_maas_cleanup.yml

```diff
@@ -2,5 +2,20 @@
 - hosts: job_nodes
   connection: local
   gather_facts: False
-  roles:
-    - { role: maas_entities, state: 'absent'}
+  tasks:
+    - name: List host entities
+      command: "raxmon-entities-list"
+      failed_when: "{{ list_entities.stdout | regex_search('Total: [0-9]+$') == None }}"
+      delegate_to: localhost
+      register: list_entities
+      until: list_entities|success
+      retries: 3
+      delay: 5
+
+    - name: Delete host entities
+      command: "raxmon-entities-delete --id {{ list_entities.stdout | regex_search(item) | regex_search('en[a-zA-Z0-9]+') | quote }}"
+      with_items: "{{ ([] | zip_longest(entity_labels, fillvalue='id=en[a-zA-Z0-9]+ label=') | map('join', '') | list) }}"
```

Comment:
Could this be rewritten as follows? Or have I missed something?
```
regex_search('id=en[a-zA-Z0-9]+ label='+item)
with_items: "{{ entity_labels }}"
```

Reply:
I've made the following change:
```diff
diff --git a/playbooks/multi_node_aio_maas_cleanup.yml b/playbooks/multi_node_aio_maas_cleanup.yml
index 545b45c..aa86ad7 100644
--- a/playbooks/multi_node_aio_maas_cleanup.yml
+++ b/playbooks/multi_node_aio_maas_cleanup.yml
@@ -13,8 +13,8 @@
       delay: 5
 
     - name: Delete host entities
-      command: "raxmon-entities-delete --id {{ list_entities.stdout | regex_search(item) | regex_search('en[a-zA-Z0-9]+') | quote }}"
-      with_items: "{{ ([] | zip_longest(entity_labels, fillvalue='id=en[a-zA-Z0-9]+ label=') | map('join', '') | list) }}"
+      command: "raxmon-entities-delete --id {{ list_entities.stdout | regex_search('id=en[a-zA-Z0-9]+ label=' + item + ' ') | regex_search('en[a-zA-Z0-9]+') | quote }}"
+      with_items: "{{ entity_labels }}"
       register: delete_entities
       until: delete_entities|success
       retries: 3
```